iis-wf/iis-wf-referenceextraction/src/test/resources/eu/dnetlib/iis/wf/referenceextraction/patent/data/retriever/report.json

```diff
@@ -1,3 +1,3 @@
 {"key":"processing.referenceExtraction.patent.retrieval.fromCache.total", "type":"COUNTER", "value": "0"}
-{"key":"processing.referenceExtraction.patent.retrieval.processed.total", "type":"COUNTER", "value": "2"}
+{"key":"processing.referenceExtraction.patent.retrieval.processed.total", "type":"COUNTER", "value": "1"}
```

Comment:
Hmm, not so sure about this.
If we had one successfully processed record and one resulting in failure I would expect to have `2` among `processed.total`. Fixing the way we store cached entries (by removing empty entry) should not affect the counters and the state we expose to clients.